src/domain/pricing/editModal/editModal.module.scss

```diff
@@ -0,0 +1,31 @@
+@import 'colours';
+@import 'spacings';
+@import 'z-index';
+
+.formTitle {
+}
+
+.formTitleLabel {
+  font-size: 14px;
+  font-weight: bold;
+}
+
+.formTitleValue {
+  margin-top: units(0.25);
+  font-size: 18px;
+}
+
+.row {
+  margin: units(2) 0;
+
+  > *:not(:first-child) {
+    margin-left: units(1);
+  }
+}
+
+.buttonRow {
+  margin-top: units(4);
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
Style has no effect

Reply:
Fixed. This also made me realize that this file should be called `editForm` instead, and half of these styles belong to `FormTypeTitle` so I put them in a separate module.